// @awesome.me/webawesome/dist-cdn/webawesome.js
/*! Copyright 2026 Fonticons, Inc. - https://webawesome.com/license */
import "./chunks/chunk.VWU24OZA.js";
import "./chunks/chunk.N2TXQSKF.js";
import "./chunks/chunk.KBS6YHTA.js";
import {
  discover,
  preventTurboFouce,
  startLoader,
  stopLoader
} from "./chunks/chunk.2PWIIYRH.js";
import {
  allDefined
} from "./chunks/chunk.X55YNZ3B.js";
import {
  serialize
} from "./chunks/chunk.LRYJ2M5H.js";
import "./chunks/chunk.I3SFSSFT.js";
import "./chunks/chunk.2LCUCOAP.js";
import {
  registerTranslation
} from "./chunks/chunk.HPOJN4W7.js";
import {
  getDefaultIconFamily,
  registerIconLibrary,
  setDefaultIconFamily,
  unregisterIconLibrary
} from "./chunks/chunk.IVRNYRHD.js";
import "./chunks/chunk.KPN3YZ6U.js";
import "./chunks/chunk.F5JLNOSF.js";
import {
  getBasePath,
  getKitCode,
  setBasePath,
  setKitCode
} from "./chunks/chunk.K6QMUIHP.js";
import {
  getAnimationNames,
  getEasingNames
} from "./chunks/chunk.OJH6FSGP.js";
import "./chunks/chunk.AIIMJL75.js";
export {
  allDefined,
  discover,
  getAnimationNames,
  getBasePath,
  getDefaultIconFamily,
  getEasingNames,
  getKitCode,
  preventTurboFouce,
  registerIconLibrary,
  registerTranslation,
  serialize,
  setBasePath,
  setDefaultIconFamily,
  setKitCode,
  startLoader,
  stopLoader,
  unregisterIconLibrary
};
