// @awesome.me/webawesome/dist/webawesome.js
/*! Copyright 2026 Fonticons, Inc. - https://webawesome.com/license */
import "./chunks/chunk.VWU24OZA.js";
import "./chunks/chunk.N2TXQSKF.js";
import "./chunks/chunk.KBS6YHTA.js";
import {
  discover,
  preventTurboFouce,
  startLoader,
  stopLoader
} from "./chunks/chunk.2PWIIYRH.js";
import {
  allDefined
} from "./chunks/chunk.X55YNZ3B.js";
import {
  serialize
} from "./chunks/chunk.LRYJ2M5H.js";
import {
  registerTranslation
} from "./chunks/chunk.OKXBNRE6.js";
import "./chunks/chunk.72WJND5X.js";
import {
  getDefaultIconFamily,
  registerIconLibrary,
  setDefaultIconFamily,
  unregisterIconLibrary
} from "./chunks/chunk.IVRNYRHD.js";
import "./chunks/chunk.KPN3YZ6U.js";
import "./chunks/chunk.F5JLNOSF.js";
import {
  getBasePath,
  getKitCode,
  setBasePath,
  setKitCode
} from "./chunks/chunk.K6QMUIHP.js";
import {
  getAnimationNames,
  getEasingNames
} from "./chunks/chunk.46TQDRI6.js";
import "./chunks/chunk.7VGCIHDG.js";
export {
  allDefined,
  discover,
  getAnimationNames,
  getBasePath,
  getDefaultIconFamily,
  getEasingNames,
  getKitCode,
  preventTurboFouce,
  registerIconLibrary,
  registerTranslation,
  serialize,
  setBasePath,
  setDefaultIconFamily,
  setKitCode,
  startLoader,
  stopLoader,
  unregisterIconLibrary
};
